tests/test_aiohttp.py

```diff
@@ -37,7 +37,8 @@ def test_fixed_host_header() -> None:
     assert _fixed_host_header("http://[fe80::1]:8000/root%desc") == {}
 
 
-def test_server_init() -> None:
+@pytest.mark.asyncio
+async def test_server_init() -> None:
```

Comment:
Why the async?

Reply:
This test creates an `AiohttpNotifyServer` which calls `asyncio.get_event_loop()` in it's `__init__`. The `get_event_loop` function wants a running event loop and as of Python 3.10 [emits a deprecation warning](https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.get_event_loop) if there isn't one. The `pytest.mark.asyncio` decorator creates the loop, and I wasn't sure if the test function needed to be `async` for that to work.

Reply:
Reading the documentation for [pytest-asyncio](https://pypi.org/project/pytest-asyncio/), it looks like I could use an event_loop fixture, if that's what you'd prefer?

Reply:
This is fine.